pkgs/development/libraries/libprom/fix-install-dir.patch

```diff
@@ -0,0 +1,10 @@
+diff -Naur prometheus-client-c-0.1.1.old/prom/CMakeLists.txt prometheus-client-c-0.1.1.new/prom/CMakeLists.txt
+--- prometheus-client-c-0.1.1.old/prom/CMakeLists.txt	2020-06-18 01:03:10.133059027 +0000
++++ prometheus-client-c-0.1.1.new/prom/CMakeLists.txt	2020-06-18 01:03:43.961228664 +0000
+@@ -137,4 +137,4 @@
+ 
+ include(CPack)
+ install(TARGETS prom ARCHIVE)
+-install(DIRECTORY include/ DESTINATION /usr/include/)
+\ No newline at end of file
++install(DIRECTORY include/ DESTINATION usr/include)
```

Comment:
```suggestion
+install(DIRECTORY include/ DESTINATION include)
```

or better with `GNUInstallDirs`

```suggestion
+install(DIRECTORY include/ DESTINATION ${CMAKE_INSTALL_FULL_INCLUDEDIR})
```

Reply:
Done (the one without /usr ). I like the GNUInstallDirs option though, I'll update my PR to digitalocean.

Reply:
I think it would be best to update the PR and then fetch the patch from there using `fetchpatch` (e.g. https://github.com/digitalocean/prometheus-client-c/commit/245c14aafde1dc015c6467f8145b5ffc4c3dbcf6.patch).

Reply:
@jtojnar (sorry for the repeat message - just adding to the right conversation). That patch doesn't apply to 0.1.1.